File: CHL-Frontend/src/app/model/Doctor.ts
export class Doctor {
    constructor(
      public id: number,
      public name: string,
      public aadhaarNo: number,
      public registrationNo: number,
      public yearOfRegistration: number,
      public state: string,
      public qualification: string,
      public email: string,
      public contact: number,
      public username: string,
      public password: string
    ){}
  }

File: CHL-Backend/src/main/java/com/CentralHealthLog/entity/Doctor.java
package com.CentralHealthLog.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Doctor {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(unique=true,nullable = false)
    private String username;

    @Column(nullable = false)
    private String password;

    @Column(nullable = false)
    private String doctorName;

    private Integer registrationNo;

    private Integer yearOfReg;

    private String stateMedicalCouncil;

    private String qualification;

    @Column(unique = true)
    private String email;

    @Column(unique=true,nullable = false)
    private String phoneNo;

    private String speciality;

    private Integer age;

    private String gender;

    private String address;

    private String statusCode;
}
